src/widgets/ConfirmationDialog.tsx

```diff
@@ -90,7 +91,7 @@ export default React.memo(function ConfirmationDialog(props: PropsType) {
           visible={props.visible}
           onDismiss={props.onCancel}
           dismissable={props.dismissable && !loading}
-          style={{ maxHeight: "100%" }}
+          style={{ maxHeight: "100%", alignSelf: "center", width: widescreen ? 560 : 280 }}
```

Comment:
Maybe it should be maxWidth instead of width? (same for below)

Reply:
Do you mean the dialog should use the maximum width it can, or do you mean it should use the minimum width until it reaches maxWidth?

Reply:
I mean replacing `width` here with `maxWidth`, so that it's at most 560/280 and not always (e.g. think smaller screens).

Reply:
Then I'd reather leave width like that and set `maxWidth:100%` because it looks nicer. If I don't set the width it looks cramped up in a tiny dialog.

Reply:
You're 100% right, go with that.